source/eitri/eitri.py

```diff
@@ -67,9 +67,9 @@ def eitri(
         sys.path.append(p)
 
     from ed_handler import generate_nav
-    from docker_wrapper import PostgresDocker
+    from navitiacommon.docker_wrapper import PostgisDocker
```

Comment:
It might be just me but I don't like the idea of deploying test's code in production.

Reply:
It's now been moved to its own `navitiacommon/tests` separate package, properly ignored by `setuptools` ([here](https://github.com/CanalTP/navitia/pull/2863/files?file-filters%5B%5D=.py&hide-deleted-files=true#diff-ddb54e68e825b008235cb85af960e635R40)) (I insist, **PROPERLY** ignored :nauseated_face: ).